profiles/multisite_drupal_communities/multisite_drupal_communities.info

```diff
@@ -115,3 +115,6 @@ dependencies[] = multisite_autosave
 dependencies[] = multisite_custom_error
 dependencies[] = multisite_settings_core
 dependencies[] = multisite_wysiwyg
+
+; NextEuropa features.
+dependencies[] = nexteuropa_laco
```

Comment:
nexteuropa_laco has a dependency to nexteuropa_multilinual . Adding a dependency to Laco here means that building a clean installation of a Community profile result in a broken installation. More details at NEPCB-30